src/main/kotlin/no/item/gradle/InitRunnerTask.kt

```diff
@@ -0,0 +1,22 @@
+package no.item.gradle
+
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+import java.io.File
+import javax.inject.Inject
+
+open class InitRunnerTask @Inject constructor(private val extension: InitRunnerExtension) : DefaultTask(){
+
+    @TaskAction
+    fun findXmlFiles() {
+        val list: MutableList<String> = ArrayList()
+        File(extension.fileDir).walk().forEach {
+            if (it.isFile&&it.toString().endsWith(".xml")){
+                list.add(it.absolutePath);
+            }
+        }
+        println("Found XML files: ")
```

Comment:
We can keep this for now. But later I suggest we only do `println` to indicate that we generate a new file, or that we update an existing file.

Reply:
Yeah, this kind of printing will be removed once the code is out of infancy.